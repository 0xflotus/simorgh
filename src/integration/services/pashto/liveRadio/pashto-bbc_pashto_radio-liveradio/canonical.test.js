/**
 * @service pashto
 * @pathname /pashto/bbc_pashto_radio/liveradio
 */

import runCanonicalTests from '../../../../pages/liveRadio/canonicalTests';

describe('Canonical pashto liveRadio', runCanonicalTests);
