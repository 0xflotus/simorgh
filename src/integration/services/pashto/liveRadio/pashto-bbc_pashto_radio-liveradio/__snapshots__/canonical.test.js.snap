// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical pashto liveRadio A11y I can see the skip to content link 1`] = `"مطلب ته ورشئ"`;

exports[`Canonical pashto liveRadio Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=68&amp;p=pashto.bbc_pashto_radio.liveradio.page&amp;x1=[urn:bbc:ares:ws_media:live:bbc_pashto_radio]&amp;x2=[responsive]&amp;x3=[news-pashto]&amp;x4=[ps]&amp;x7=[player-live]&amp;x8=[simorgh-nojs]&amp;x9=[بي+بي+سي+افغانستان+(پښتو+خپرونه)+-+BBC+News+پښتو]\\"/>"`;

exports[`Canonical pashto liveRadio Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/pashto"`;

exports[`Canonical pashto liveRadio Footer I can see the footer copyright and external linking text 1`] = `"© 2020 بي بي سي. بي بي‌ سي‌ د نورو ویبپاڼو د محتوا مسوله نه ده. د نورو ویبپاڼو لینکولو په اړه زموږ تګلاره."`;

exports[`Canonical pashto liveRadio Header I can see the branding 1`] = `"BBC News, پښتو"`;

exports[`Canonical pashto liveRadio I can see the headline 1`] = `"بي بي سي افغانستان (پښتو خپرونه)"`;

exports[`Canonical pashto liveRadio I can see the summary 1`] = `"بي بي سي د افغانستان لپاره کورني، سیمه ییز او نړیوال وروستي او کره خبرونه د هر اړخېزو څېړونو او شننو سره تاسې ته وړاندې کوي. په پښتو او دري ژبو بېلا بېلې سیاسي، اقتصادي، ټولنېزې او روزنیزې خپرونې هر ورځ د سهار له ۵ بجو نه د شپې تر ۱۲ بجو پورې خپروي"`;

exports[`Canonical pashto liveRadio Media player embed A11y I can read the media player title 1`] = `"اډیو پلیئر"`;

exports[`Canonical pashto liveRadio Media player embed I can see the media player embed 1`] = `"https://polling.test.bbc.co.uk/ws/av-embeds/media/bbc_pashto_radio/liveradio/ps?morph_env=live"`;

exports[`Canonical pashto liveRadio SEO Canonical link 1`] = `"http://localhost:7080/pashto/bbc_pashto_radio/liveradio"`;

exports[`Canonical pashto liveRadio SEO Dir attribute 1`] = `"rtl"`;

exports[`Canonical pashto liveRadio SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical pashto liveRadio SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical pashto liveRadio SEO Heading level 1 1`] = `"بي بي سي افغانستان (پښتو خپرونه)"`;

exports[`Canonical pashto liveRadio SEO Lang attribute 1`] = `"ps"`;

exports[`Canonical pashto liveRadio SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "RadioChannel",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pashto.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/pashto/bbc_pashto_radio/liveradio",
    "@type": "WebPage",
    "name": "بي بي سي افغانستان (پښتو خپرونه)",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/pashto.png",
      "width": 1024,
    },
    "name": "BBC News پښتو",
    "publishingPrinciples": "https://www.bbc.com/pashto/institutional-49283007",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/pashto.png",
  "url": "http://localhost:7080/pashto/bbc_pashto_radio/liveradio",
}
`;

exports[`Canonical pashto liveRadio SEO OG description 1`] = `"بي بي سي د افغانستان لپاره کورني، سیمه ییز او نړیوال وروستي او کره خبرونه د هر اړخېزو څېړونو او شننو سره تاسې ته وړاندې کوي. په پښتو او دري ژبو بېلا بېلې سیاسي، اقتصادي، ټولنېزې او روزنیزې خپرونې هر ورځ د سهار له ۵ بجو نه د شپې تر ۱۲ بجو پورې خپروي"`;

exports[`Canonical pashto liveRadio SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/pashto.png"`;

exports[`Canonical pashto liveRadio SEO OG image alt 1`] = `"BBC News پښتو"`;

exports[`Canonical pashto liveRadio SEO OG locale 1`] = `"ps"`;

exports[`Canonical pashto liveRadio SEO OG site name 1`] = `"BBC News پښتو"`;

exports[`Canonical pashto liveRadio SEO OG title 1`] = `"بي بي سي افغانستان (پښتو خپرونه) - BBC News پښتو"`;

exports[`Canonical pashto liveRadio SEO OG type 1`] = `"website"`;

exports[`Canonical pashto liveRadio SEO OG url 1`] = `"http://localhost:7080/pashto/bbc_pashto_radio/liveradio"`;

exports[`Canonical pashto liveRadio SEO Page title 1`] = `"بي بي سي افغانستان (پښتو خپرونه) - BBC News پښتو"`;

exports[`Canonical pashto liveRadio SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical pashto liveRadio SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical pashto liveRadio SEO Twitter creator 1`] = `"@bbcpashto"`;

exports[`Canonical pashto liveRadio SEO Twitter description 1`] = `"بي بي سي د افغانستان لپاره کورني، سیمه ییز او نړیوال وروستي او کره خبرونه د هر اړخېزو څېړونو او شننو سره تاسې ته وړاندې کوي. په پښتو او دري ژبو بېلا بېلې سیاسي، اقتصادي، ټولنېزې او روزنیزې خپرونې هر ورځ د سهار له ۵ بجو نه د شپې تر ۱۲ بجو پورې خپروي"`;

exports[`Canonical pashto liveRadio SEO Twitter image alt 1`] = `"BBC News پښتو"`;

exports[`Canonical pashto liveRadio SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/pashto.png"`;

exports[`Canonical pashto liveRadio SEO Twitter site 1`] = `"@bbcpashto"`;

exports[`Canonical pashto liveRadio SEO Twitter title 1`] = `"بي بي سي افغانستان (پښتو خپرونه) - BBC News پښتو"`;
